### PR TITLE
Razor remove csproj

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review and merge into csproj branch

File: aspnetcore/mvc/views/razor.md [hunk 1]
```diff
@@ -569,17 +567,12 @@ C# Razor keywords need to be double escaped with `@(@C# Razor Keyword)`, for exa
 
 ## View compilation
 
-Razor views are compiled at runtime when the view is invoked. If you prefer to compile your Razor views and deploy them with your app make these changes to *project.json*:
+Razor views are compiled at runtime when the view is invoked. ASP.NET Core 1.1.0 and higher can optionally compile Razor views and deploy them with the app. To do this you must set `MvcRazorCompileOnPublish` to true and include a package reference to `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation`. The following *.csproj* sample highlights these settings:
```
Comment (tdykstra) [inline aspnetcore/mvc/views/razor.md]: Where do you set MvcRazorCompileOnPublish?  Or is that already shown in unchanged parts of the doc?

Comment (tdykstra) [inline aspnetcore/mvc/views/razor.md]: Also, you could drop "you must"

**Comment by Rick-Anderson** [comment]
It's highlighted in the ._csproj_ file I added.

 <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>

**Pull request merged**